README.md

```diff
@@ -1,75 +1,69 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio website
 
-> Description the project.
+## Project Description .
 
+A personal portfolio site of to showcase my skills and what i am capable of to potential employers.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+-Html, Css
 
-## Live Demo (if available)
+## $Getting Started
 
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+To get a local copy up and running follow these simple steps.
 
 ### Prerequisites
 
-### Setup
-
-### Install
-
-### Usage
+- a computer running any operating system.
+- an internet connection
+- knowledge of basic git commands
 
-### Run tests
+### Setup
 
-### Deployment
+- link your computer to github
+- clone the repository
+- open the file in your code editor
+- set up Html and css linters for the project
```

Comment:
You can use this 👇  as a guide on how to write the setup and prerequisite
![readme](https://user-images.githubusercontent.com/73854999/198215840-2d9835a1-1d43-4a02-9b61-993b027ee5e6.PNG)